selftests/unit/test_datadir.py

```diff
@@ -34,6 +34,54 @@ def _get_temp_dirs_mapping_and_config(self):
         config_file.close()
         return (base_dir, mapping, config_file.name)
 
+    @staticmethod
+    def _make_dummy_registrations(stg):
+        """
+        Register dummy options to be updated by the new Settings API
+
+        The new Settings API never register options from config files. This
+        method creates dummy registrations to be updated later with the
+        correct values for the options.
+        """
+        help_msg = 'Cache directories to be used by the avocado test'
+        stg.register_option(section='datadir.paths',
+                            key='cache_dirs',
+                            key_type=list,
+                            default=[],
+                            help_msg=help_msg)
+
+        help_msg = 'Base directory for Avocado tests and auxiliary data'
+        stg.register_option(section='datadir.paths',
+                            key='base_dir',
+                            key_type=os.path.expanduser,
+                            default=os.path.expanduser(
+                                '/var/lib/avocado'),
+                            help_msg=help_msg)
+
+        help_msg = 'Test directory for Avocado tests'
+        stg.register_option(section='datadir.paths',
+                            key='test_dir',
+                            key_type=os.path.expanduser,
+                            default=os.path.expanduser(
```

Comment:
What's the point of using `os.path.expanduser` with a literal string that won't be expanded?

Reply:
Leftover from the development phase. I'll update it.